test/System.IdentityModel.Tokens.Jwt.Tests/CreateAndValidateTokens.cs

```diff
@@ -25,13 +25,18 @@
 //
 //------------------------------------------------------------------------------
 
-using System;
 using System.Collections.Generic;
 using System.Globalization;
-using System.IdentityModel.Tokens.Tests;
 using System.Security.Claims;
+using Microsoft.IdentityModel.Tokens;
+using Microsoft.IdentityModel.Tokens.Tests;
 using Xunit;
 
+// since we are in the System ns, we need to map to M.IM.Tokens
+using SigningCreds = Microsoft.IdentityModel.Tokens.SigningCredentials;
```

Comment:
As I hoped the comment indicated, the compiler will default to System.IdentityModel.Tokens.SigningCredentials. So we have to explicitly define the type.